Boot.st

```diff
@@ -83,8 +83,7 @@ devsesh onPreSaveImage.
 	maxObjects: 24576*1024
 ] ensure: [devsesh onPostSaveImage]!
 
-"Remove rogue .img and unnecessary .chg files"
-File delete: (File default: SessionManager current imagePath extension: 'img').
```

Comment:
The cause of the rogue .img is the incorrect return value of `BootSessionManager class>>imageExtension`:
* Boot.st line 58 saves the image to its intended new filename just before invoking DolphinProduct>>boot.
* Then see BootSessionManager>>saveImageTo:, where (after updating the value of `imagePath`) it calls `self snapshot: self imageFileName`.
* #imageFileName constructs its answer from the answers to #imagePath and #imageExtension, and #imageExtension simply forwards class-side.

As far as `SessionManager current` being a DevelopmentSessionManager by this point in the boot process: This is true, but all the script cared about at that point is that it had the correct value for `imagePath`, which of course it did, having copied it from the BootSessionManager as part of #installNew.